psp/tests/data/data_sources/test_pv_data_source.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta
+
+import numpy as np
+import pytest
+import xarray as xr
+
+from psp.data.data_sources.pv import NetcdfPvDataSource
+
+
+@pytest.fixture
+def pv_data_source(tmp_path):
```

Comment:
worth moving this to a conftest.py file?

Reply:
Not yet! When multiple test files can use the same `NetcdfPvDataSource` then yes!